### PR TITLE
[WIP] RFC: Deletion and deprecation

### PR DESCRIPTION
**Pull request by hannes-ucsc**

### CODE REVIEW

File: specs/data-store/deletion-and-deprecation.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+versions. Administrators are responsible for tracking down data bundles with
+data derived from data in deleted bundles and to issue deletion requests for
+them.
+
+### Tiered storage
+
+Physical deletion of blobs is effective across all tiered storage classes in
+the underlying blob store (S3, GCP, ...).
+
+TBD: Does deletion from Glacier involve a delay?
```
Comment (kislyuk) [inline specs/data-store/deletion-and-deprecation.md]: It does not, but we don't currently plan to use Glacier. (We plan to use S3-IA only for cost control on S3)

File: specs/data-store/deletion-and-deprecation.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+           - physical
+           title: Describes the action to take on files in the deleted bundle.
+           description: |
+              - `logical`: Remove the metadata files in this bundle from any 
+                           index and make the data files inaccessible but do not 
+                           physically delete it. Remove or make inaccessible any 
+                           secondary copies of those files.
+              - `physical`: Physically and irreversibly delete the data and 
+                            metadata files in the bundle, and any copy thereof.
+         contact:
```
Comment (hannes-ucsc) [inline specs/data-store/deletion-and-deprecation.md]: Add description

**Comment by parthshahva** [comment]
@hannes-ucsc can this PR be closed?

**Comment by hannes-ucsc** [comment]
@parthshahva, fine by me, but @Bento007 asked me about this PR recently. You should confirm with him.

**Pull request closed without merging**